lib/browser/api/web-contents.ts

```diff
@@ -539,7 +539,10 @@ WebContents.prototype._init = function () {
     value: {
       getActiveIndex: this._getActiveIndex.bind(this),
       length: this._historyLength.bind(this),
-      getEntryAtIndex: this._getNavigationEntryAtIndex.bind(this)
+      getEntryAtIndex: this._getNavigationEntryAtIndex.bind(this),
+      deleteEntryAtIndex: this._deleteNavigationEntryAtIndex.bind(this),
+      getHistory: this._getHistory.bind(this),
+      replaceHistory: this._replaceHistory.bind(this)
```

Comment:
What do you think of renaming to just `navigationHistory.get()` and `navigationHistory.replace()` to reduce redundancy

Reply:
You're right, that would be nicer. I renamed them.